site/content/blog/2021-08-20-zap-fileupload-addon/index.md

```diff
@@ -47,10 +47,10 @@ Vulnerabilities covered are:
 - Remote Code Execution by uploading JSP and its variants
 - No virus scan. 
 
-The FileUpload add-on is currently only available for dev versions of ZAP such as weekly releases or built from source, as it depends on an up-to-date version of ZAP's core.
+The FileUpload add-on is available in ZAP Marketplace.
```

Comment:
"in the" or "via the" - cpuld link "ZAP Marketplace" to https://www.zaproxy.org/addons/

Reply:
I'd suggest keeping the previous sentence and add one that mentions the add-on will be available in the marketplace for newer ZAP versions.

Reply:
I believe thc202's comment is still outstanding.

Reply:
To be clear I'm suggesting that to not rewrite history, the blog posts contents are accurate on/for the date they were written and one should read them taking into account that.